controllers/orderController.js

```diff
@@ -0,0 +1,36 @@
+const orderService = require('../services/orderService');
+const userService = require('../services/userService');
+const { catchAsync } = require('../middlewares/error');
+
+const getUserInfo = catchAsync(async (req, res) => {
+  const userId = req.user.id;
+
+  if (!userId) {
+    const error = new Error('KEY_ERROR');
+    error.statusCode = 400;
+    throw error;
+  }
+
+  const user = await userService.getUserById(userId);
+
+  return res.status(200).json({ user });
+});
+
+const completeOrders = catchAsync(async (req, res) => {
+  const user = req.user;
+  const { address, SubscribeDeliveryTime, cartIds } = req.body;
```

Comment:
subscribeDeliveryTime 왜 이 변수만 앞이 대문자로 시작하죠?

```suggestion
  const { address, subscribeDeliveryTime, cartIds } = req.body;
```